pegasus-spark-analyser/src/main/java/com/xiaomi/infra/pegasus/spark/analyser/ColdBackupConfig.java

```diff
@@ -44,4 +57,31 @@ public ColdBackupConfig setDataVersion(DataVersion dataVersion) {
     this.dataVersion = dataVersion;
     return this;
   }
+
+  /**
+   * the max file open count
+   *
+   * @param dbMaxFileOpenCount dbMaxFileOpenCount is rocksdb concept which can control the max file
+   *     open count, default is 50. detail see
+   *     https://github.com/facebook/rocksdb/wiki/RocksDB-Tuning-Guide#general-options
+   * @return this
+   */
+  public ColdBackupConfig setDbMaxFileOpenCount(int dbMaxFileOpenCount) {
```

Comment:
```suggestion
  public ColdBackupConfig setMaxOpenFiles(int maxOpenFiles) {
```

The user of ColdBackupConfig should not be aware of "RocksDB".
RocksDB is the underlying implementation details that we need to hide from upper-level users.
We probably change the storage engine someday, but the users need only know "maxOpenFiles", without knowing "DB".